web/src/main/java/edu/softserve/zoo/controller/rest/impl/EmployeeRestController.java

```diff
@@ -31,4 +34,10 @@ public EmployeeRestController() {
     protected Service<Employee> getService() {
         return employeeService;
     }
+
+    @Override
+    @RequestMapping(method = RequestMethod.GET)
+    public List<EmployeeDto> getAll() {
+        return super.getAll();
```

Comment:
you are returning ALL employees. do you REALLY need to return ALL employees? If so, name these cases please